Drop-In/src/sharedTest/java/com/braintreepayments/api/MockBraintreeClientBuilder.java

```diff
@@ -97,87 +69,42 @@ public MockBraintreeClientBuilder integration(String integration) {
         return this;
     }
 
-    public MockBraintreeClientBuilder returnUrlScheme(String returnUrlScheme) {
-        this.returnUrlScheme = returnUrlScheme;
-         return this;
-    }
-
-    public MockBraintreeClientBuilder urlSchemeDeclaredInManifest(boolean urlSchemeInAndroidManifest) {
-        this.urlSchemeInAndroidManifest = urlSchemeInAndroidManifest;
-        return this;
-    }
-
-    public MockBraintreeClientBuilder canPerformBrowserSwitch(boolean canPerformBrowserSwitch) {
-        this.canPerformBrowserSwitch = canPerformBrowserSwitch;
-        return this;
-    }
-
     public BraintreeClient build() {
         BraintreeClient braintreeClient = mock(BraintreeClient.class);
         when(braintreeClient.getAuthorization()).thenReturn(authorization);
         when(braintreeClient.getSessionId()).thenReturn(sessionId);
         when(braintreeClient.getIntegrationType()).thenReturn(integration);
-
-        // HACK: some google pay tests fail when getReturnUrlScheme is stubbed but not invoked
-        // TODO: create a wrapper around google wallet api to avoid having to use Powermock and Robolectric at the same time, which seems to be causing this issue
-        if (returnUrlScheme != null) {
-            when(braintreeClient.getReturnUrlScheme()).thenReturn(returnUrlScheme);
-        }
-
-        when(braintreeClient.isUrlSchemeDeclaredInAndroidManifest(anyString(), any(Class.class))).thenReturn(urlSchemeInAndroidManifest);
-        when(braintreeClient.canPerformBrowserSwitch(any(FragmentActivity.class), anyInt())).thenReturn(canPerformBrowserSwitch);
-        when(braintreeClient.getManifestActivityInfo(any(Class.class))).thenReturn(activityInfo);
-
-        doAnswer(new Answer<Void>() {
-            @Override
-            public Void answer(InvocationOnMock invocation) {
-                ConfigurationCallback callback = (ConfigurationCallback) invocation.getArguments()[0];
-                if (configuration != null) {
-                    callback.onResult(configuration, null);
-                } else if (configurationError != null) {
-                    callback.onResult(null, configurationError);
-                }
-                return null;
+//        when(braintreeClient.isUrlSchemeDeclaredInAndroidManifest(anyString(), any(Class.class))).thenReturn(true);
+//        when(braintreeClient.canPerformBrowserSwitch(any(FragmentActivity.class), anyInt())).thenReturn(true);
```

Comment:
nit: good to remove these? I guess we can always check git history if needed.

Reply:
Ah yes meant to remove that good catch!